From 84a300f96343825fdf289bb452f9aea17a7f1a31 Mon Sep 17 00:00:00 2001
From: Tim Gardner <tim.gardner@canonical.com>
Date: Mon, 13 Feb 2012 14:19:58 -0700
Subject: [PATCH 3/5] Add support for 3.x kernel versions

As suggested by Kano on Freenode #ubuntu-kernel

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

---

Taken from git://anonscm.debian.org/collab-maint/ndiswrapper.git
debian/patches/Add-support-for-3.x-kernel-versions.patch

Signed-off-by: Andrey Utkin <andrey.krieger.utkin@gmail.com>
---
 utils/ndiswrapper | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/ndiswrapper b/utils/ndiswrapper
index 4bb91f2..ba082a3 100755
--- a/utils/ndiswrapper
+++ b/utils/ndiswrapper
@@ -55,7 +55,7 @@ if (@ARGV < 1) {
 
 my $modconf;
 if (`uname -r` =~ /(\d+)\.(\d+)\.(\d+)/) {
-    if ($2 > 4) {
+    if (($2 > 4) || ($1 > 2)) {
 	if (-d "/etc/modprobe.d") {
 	    $modconf = "/etc/modprobe.d/ndiswrapper.conf";
 	} else {
-- 
1.8.4

